backend/src/nodes/pytorch_nodes.py

```diff
@@ -4,10 +4,12 @@
 
 from __future__ import annotations
 
+from io import BytesIO
 import os
 from typing import Any, OrderedDict, Union
 
 import numpy as np
+import onnx
```

Comment:
Onnx is an optional dependency. Just importing it here will cause the backend to crash for all people (e.g. me) that have PyTorch installed but not Onnx.

Reply:
It's actually an unused import from a previous approach that ended up being unnecessary, just forgot to remove the import.